### PR TITLE
add smartphone-charging icon

### PR DESCRIPTION
**Pull request by mittalyashu**
| Preview |
| --- |
| CleanShot 2021-05-09 at 17 33 21@2x |

### CODE REVIEW

**Comment by mittalyashu** [comment]
@ericfennis LGTM

**Comment by locness3** [comment]
`smartphone-zap`, please.

**Comment by mittalyashu** [comment]
@locness3 We already have similar name with charging.

CleanShot 2021-05-15 at 14 28 55@2x

**Comment by mittalyashu** [comment]
@ericfennis What do you think?

File: icons/smartphone-charging.svg [hunk 1]
```diff
@@ -0,0 +1,14 @@
...
+  width="24"
+  height="24"
+  viewBox="0 0 24 24"
+  fill="none"
+  stroke="currentColor"
+  stroke-width="2"
+  stroke-linecap="round"
+  stroke-linejoin="round"
+>
+  <path d="M17 2H7a2 2 0 00-2 2v16a2 2 0 002 2h10a2 2 0 002-2V4a2 2 0 00-2-2z" />
```
Comment (ericfennis) [inline icons/smartphone-charging.svg:12]: I think it's good!
Small improvement

```suggestion
  <rect x="5" y="2" width="14" height="20" rx="2" ry="2" />
```

**Review by ericfennis — APPROVED** [review]

**Pull request merged**